test/unit/services/query-service.test.js

```diff
@@ -119,5 +119,214 @@ describe('query-service', () => {
       expect(validatedResponseNoContext).toStrictEqual(false)
       expect(validatedResponseNoAnswer).toStrictEqual(false)
     })
+
+    test('fetchAnswer returns summary answer when summariesEnabled is true', async () => {
+      createRetrievalChain.mockResolvedValue({
+        invoke: jest.fn().mockResolvedValue({
+          answer: JSON.stringify({ answer: 'summary response', items: [] })
```

Comment:
This mock is used by both full and summary branches of the code. So even the full index will return "summary response". Usually we'd refactor the code and put `runFetchAnswerQuery` in its own file, mock it and check if its being called with summariesMode set to false (expecting false). 

Instead of refactoring there's a quicker option, see below...